src/main/java/mapWithNameAndPassword/CustomerNameAndPass.java

```diff
@@ -0,0 +1,34 @@
+package mapWithNameAndPassword;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Temporary class for store customers names and passwords
+ */
+public class CustomerNameAndPass {
+
+    public static Map<String, String> map;
```

Comment:
this map must be private and you must interact with it by CustomerNameAndPass methods

Reply:
Done